src/transformers/trainer.py

```diff
@@ -521,6 +521,9 @@ def __init__(
                 if isinstance(model, PreTrainedModel):
                     # Patch the model with liger kernels. Use the default kernel configurations.
                     _apply_liger_kernel_to_instance(model=model)
+                elif hasattr(model.get_base_model) and isinstance(model.get_base_model(), PreTrainedModel):
+                    # Patch the base model with liger kernels where model is a PeftModel. Use the default kernel configurations.
+                    _apply_liger_kernel_to_instance(model=model.base_model.model)
```

Comment:
Sorry, missed this earlier:
```suggestion
                    _apply_liger_kernel_to_instance(model=model.get_base_model())
```